Source/diablo.h

```diff
@@ -7,19 +7,19 @@
 #define __DIABLO_H__
 
 extern HWND ghMainWnd;
-extern int glMid1Seed[NUMLEVELS];
-extern int glMid2Seed[NUMLEVELS];
+extern int glMid1Seed[NUMLEVELS + 1];
+extern int glMid2Seed[NUMLEVELS + 1];

```

Comment:
Remember that the hellfire team tampered with the code. Despite the devs having an expensive education and all, they couldn't do basic math.
```c
//int glEndSeed[17];		// @@@ drb temp       **** old code ****
//int glMid1Seed[17];
//int glMid2Seed[17];
//int glMid3Seed[17];

int glEndSeed[NUMLEVELS+1];		// @@@ drb temp         *** hellfire's code***
int glMid1Seed[NUMLEVELS+1];
int glMid2Seed[NUMLEVELS+1];
int glMid3Seed[NUMLEVELS+1];	//JKE 7/30 adds new levels
```
They must have thought the NUMLEVELS define didn't include town, when in fact it did. So this change makes these arrays 18 instead of 17. Check the PSX symbol as well:
```
STAT type ARY INT size 68 dims 1 17 tag  name glEndSeed
STAT type ARY INT size 68 dims 1 17 tag  name glMid1Seed
STAT type ARY INT size 68 dims 1 17 tag  name glMid2Seed
STAT type ARY INT size 68 dims 1 17 tag  name glMid3Seed
```